src/MongoDB/Driver/Manager.cpp

```diff
@@ -522,6 +534,136 @@ static bool hippo_mongo_driver_manager_apply_ssl_opts(mongoc_client_t *client, c
 	return 1;
 }
 
+void command_started(const mongoc_apm_command_started_t *event)
+{
+	ObjectData *obj_context = (ObjectData*) mongoc_apm_command_started_get_context(event);
+	Class *cls = obj_context->getVMClass();
+	static Class *c_event;
+	Func  *m   = cls->lookupMethod(s_MongoDBDriverManager_dispatchCommandStarted.get());
+	Variant    result;
+	MongoDBDriverManagerData* data = Native::data<MongoDBDriverManagerData>(obj_context);
+
+	Array subscribers = obj_context->o_get(s_MongoDBDriverManager_subscribers, false, s_MongoDriverManager_className).toArray();
```

Comment:
If you change the subscriber set to use `SplObjectStorage`, I suppose this would be replaced with a call to `SplObjectStorage::count()`.